incubator/hnc/internal/validators/anchor_test.go

```diff
@@ -69,6 +69,7 @@ func TestAllowCascadingDeleteSubnamespaces(t *testing.T) {
 		{name: "set in ancestor that is not the first full namespace", acd: "a", pnm: "c", cnm: "d"},
 		{name: "unset in leaf", pnm: "d", cnm: "e"},
 		{name: "unset in non-leaf", pnm: "c", cnm: "d", fail: true},
+		{name: "unset in non-leaf but bad anchor", pnm: "bad", cnm: "d"},
```

Comment:
Probably best for `bad` to actually be a namespace in the tree, otherwise there's a chance that you could hit some other random error. Maybe use `b`, since `d` is actually a child of `c`?

Reply:
SG, updated.